### PR TITLE
fix(sidebar): Add flex-shrink 0 to comment input

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 1]
```diff
@@ -86,6 +86,7 @@
 
     .bcs-activity-feed-comment-input {
         background-color: $white;
+        flex-shrink: 0;
```
Comment (booc0mtaco) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss]: What are are the other two `flex` values here? does it inherit `flex-grow` and `flex-basis` from somewhere else already? Should we use the shorthand syntax to be more deliberate?

Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss]: Good point, let me fix it to be more clear

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 2]
```diff
@@ -86,7 +86,7 @@
 
     .bcs-activity-feed-comment-input {
         background-color: $white;
-        flex-shrink: 0;
+        flex: 0 0 auto;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss]: Does the comment box still grow/shrink as expected given min/max height set below? It definitely looks like it's meant to change size.

Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss]: Yeah it does, the flex-grow: 0 specifies that it won't take up any of the remaining space in the flex container but will still reflect it's natural height

**Pull request merged**